pss-migration-orchestration/README.md

```diff
@@ -0,0 +1,86 @@
+- Generate recommended generic exceptions for customer administrators to manage. Instead of directly creating the Kyverno PolicyExceptions, we instead generate an intermediate resource which customers can store or distribute to workload teams, and which separates the cluster admin's intent from the underlying implementation. This is the approach described in the rest of this RFC.
+
+## :exclamation: Risks
+
+1. The generated exceptions could be over-permissive. We actually expect this, and are working on a separate initiative to tighten down over-permissive exceptions in the future. We believe we can undertake this work only for purposes of migration, given that the generated exceptions will represent the current state of clusters secured with PSPs. By definition, the presence of a workload means that the resource has been approved and any current risk has been accepted by the customer, since their currently configured policy allows it. In the future, when PSS is the standard, we may not be able to make this assumption when changing policy implementations. The proposed process keeps customer admins in the loop, and they could elect to honor generated exceptions only from certain clusters, or from certain environments.
+2. This implementation could be too complicated, or we are not fast enough at implementing it. This is entirely possible. We think about this two ways. 1. There is upfront process work to be done by the customer, and without the automation recommending exceptions, it is unlikely that customers move very fast anyway. Any work done while we are implementing this capability is not wasted, and the recommendations can be used to accelerate adoption later on. 2. taking the time now to get customers directly to what we expect to be the future is preferable to trying to get them to repeat this effort again later.
+3. Probably others, we're moving pretty fast.
+
+## :feet: Implementation
+
+> [!NOTE]
+> This implementation includes new resource types outlined in a parallel RFC. We will only implement the minimum necessary to carry out the migration now.
+
+### :small_blue_diamond: Flow
+
+Example CR flow:
+![PSS orchestration diagram](pss-orchestration.jpg)
+
+The intended workflow is:
+
+- (v19.1.x) security bundle containing this implementation is installed
+- `exception-recommender` generates one `PolicyExceptionDraft` per-workload with failing policies in the `policy-exceptions` namespace
+- cluster admins retrieve the `PolicyExceptionDrafts`
+- cluster admins review the generated exceptions, and, if they accept them, store them in a git repository as Giant Swarm PolicyException resources
+  - according to their preference and security policies, customers may adopt different processes here, like distributing PolicyExceptions to workload owners
```

Comment:
are policy reports visible to the teams? I mean,  are those in the same app namespace? I am thinking what does `distributing PolicyExceptions` means

Reply:
PolicyReports are created in the namespace of the workload, but not all users are granted RBAC to access them. `distributing PolicyExceptions to workload owners` here means giving the generated PolicyExceptions to individual teams to store in their own repository instead of being managed in a central place by the platform team. We expect customers will have different processes and ownership of exceptions - for example one customer with two business units may have different risk owners in different business units.